client/core/trade.go

```diff
@@ -34,12 +34,15 @@ func (err ExpirationErr) Error() string { return string(err) }
 // A matchTracker is used to negotiate a match.
 type matchTracker struct {
 	db.MetaMatch
-	id          order.MatchID
-	failErr     error
-	refundErr   error
-	prefix      *order.Prefix
-	trade       *order.Trade
-	counterSwap asset.AuditInfo
+	id            order.MatchID
+	failErr       error
+	tickMeterer   *time.Timer
+	suspectRedeem bool
+	suspectSwap   bool
+	refundErr     error
```

Comment:
I think we should group these fields separately and describe their intend, particularly `tickerMeterer`, which if I understand correctly is a gate for "suspect" swaps to prevent them from retrying too fast while not failing them outright?